config/device_configurations/maxon.yaml

```diff
@@ -0,0 +1,34 @@
+Maxon:
```

Comment:
Why did you create an config folder? This should not be in here. Please add a maxon yaml in the examle config. 
For your specific setup you should then have an extra wrist package that contains your config.

Reply:
You are write... I created for my own config, I will remove it.